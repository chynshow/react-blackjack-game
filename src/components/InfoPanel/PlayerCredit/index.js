import React from 'react';

const PlayerCredit = ({ credit, className }) => {
  return <div className={className}>Player Credit: {credit}$</div>;
};

export default React.memo(PlayerCredit);
