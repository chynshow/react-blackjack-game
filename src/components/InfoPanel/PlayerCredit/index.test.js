import React from 'react';
import PlayerCredit from './index';
import { shallow } from 'enzyme';

it('render PlayerCredit component', () => {
  const c = shallow(<PlayerCredit />);
  expect(c).toMatchSnapshot();
});
