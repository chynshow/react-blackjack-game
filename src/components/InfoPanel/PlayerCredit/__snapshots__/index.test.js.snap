// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlayerCredit component 1`] = `
<div>
  Player Credit: 
  $
</div>
`;
